src/interface/mojaloop.yaml

```diff
@@ -270,12 +270,17 @@ paths:
           $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/501'
         503:
           $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/503'
-    delete:
+    patch:
       tags:
         - consents
-      summary: '[Mojaloop Callback] Result of deleting the specified consent'
-      description: Result of deleting the specified consent
-      operationId: deleteConsentsById
+      summary: '[Mojaloop Callback] Result of patching the specified consent (used in unlinking)'
+      description: Result of patching the specified consent (used in unlinking)
+      operationId: patchConsentsById
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: 'shared.yaml#/components/schemas/PatchConsentsByIdRequest'
```

Comment:
isn't it better to take this interface from api-snippets?

Reply:
Yup, I think it would be better that way! :-) 
iirc, the current dependency still points to my forked repo (because it was set before the `api-snippets` was published), probably can fix that as well

Reply:
make sense - I'll make a separate ticket for that - separation of concerns